src/pickup.cpp

```diff
@@ -111,21 +111,19 @@ Pickup::interact_results Pickup::interact_with_vehicle( vehicle *veh, const trip
     }
 
     switch( static_cast<options>( choice ) ) {
+
     case USE_HOTPLATE: {
-        //Will be -1 if no battery at all
-        item tmp_hotplate( "hotplate", 0 );
-        // Drain a ton of power
-        tmp_hotplate.charges = veh->drain( "battery", 100 );
-        if( tmp_hotplate.is_tool() ) {
-            const auto tmptool = dynamic_cast<const it_tool *>((&tmp_hotplate)->type);
-            if ( tmp_hotplate.charges >= tmptool->charges_per_use ) {
-                g->u.invoke_item( &tmp_hotplate );
-                tmp_hotplate.charges -= tmptool->charges_per_use;
-                veh->refill( "battery", tmp_hotplate.charges );
-            }
+        item pseudo( "hotplate" );
```

Comment:
Separate PR should factor this out, possibly as a method of `vehicle` unless any better suggestions?